connectivity/check/policy.go

```diff
@@ -69,9 +69,9 @@ func getCiliumPolicyRevision(ctx context.Context, pod Pod) (int, error) {
 	if err != nil {
 		return 0, err
 	}
-	revision, err := strconv.Atoi(strings.Trim(stdout.String(), "'\n"))
+	revision, err := strconv.Atoi(strings.Trim(stdout.String(), "'\r\n"))
```

Comment:
The fact that `ExecInPod` returns CRs is surprising, so trimming CRLFs should probably be handled as close to the source as possible. (also, it should use `bytes.Trim()`, avoids the unnecessary string conversion)

Reply:
It needs to be trimmed from the whole output if done from there. I.e., with a regexp rather than `.Trim()`. Not sure it's worth it. These places are expecting that line ending might be there, I'd rather keep these sites more generic and not assume `\r\n` is never seen.

Reply:
Now using `bytes.ReplaceAll()` if needed in `exec.go`.